config.json

```diff
@@ -821,7 +821,13 @@
         "name": "Phone Number",
         "uuid": "82775adb-eabe-4d44-91f5-4080b8834a4a",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "strings",
+          "arrays",
```

Comment:
Can you say a sentence why you think arrays are needed as prerequisite?

Everything else looks good to me.

Reply:
Oh, I had actually used arrays in my solution but now that I see it, it can be done without arrays. I will remove it.